crates/router/src/connector/paypal.rs

```diff
@@ -1534,9 +1533,9 @@ impl api::IncomingWebhook for Paypal {
                 Ok(api_models::webhooks::ObjectReferenceId::PaymentId(
                     api_models::payments::PaymentIdType::PaymentAttemptId(
                         resource
-                            .dispute_transactions
+                            .disputed_transactions
                             .first()
-                            .map(|transaction| transaction.reference_id.clone())
+                            .map(|transaction| transaction.seller_transaction_id.clone())
```

Comment:
If `seller_transaction_id` is not our id, we should change PaymentIdType to ConnectorTransactionId instead of PaymentAttemptId

Reply:
This should be seller_transaction_id